reladomographql/src/main/java/com/gs/reladomo/graphql/FilterQueryBuilder.java

```diff
@@ -0,0 +1,386 @@
+        DOUBLE_OPS.put ("greaterThanEquals", (attr, val) -> attr.greaterThanEquals ((double) val));
+        DOUBLE_OPS.put ("lessThan", (attr, val) -> attr.lessThan ((double) val));
+        DOUBLE_OPS.put ("lessThanEquals", (attr, val) -> attr.lessThanEquals ((double) val));
+
+        FLOAT_OPS.put ("eq", (attr, val) -> attr.eq ((float) val));
+        FLOAT_OPS.put ("notEq", (attr, val) -> attr.notEq ((float) val));
+        FLOAT_OPS.put ("isNull", (attr, val) -> attr.isNull ());
+        FLOAT_OPS.put ("isNotNull", (attr, val) -> attr.isNull ());
+        FLOAT_OPS.put ("in", (attr, val) -> attr.in (asFloatSet ((List) val)));
+        FLOAT_OPS.put ("notIn", (attr, val) -> attr.notIn (asFloatSet ((List) val)));
+        FLOAT_OPS.put ("greaterThan", (attr, val) -> attr.greaterThan ((float) val));
+        FLOAT_OPS.put ("greaterThanEquals", (attr, val) -> attr.greaterThanEquals ((float) val));
+        FLOAT_OPS.put ("lessThan", (attr, val) -> attr.lessThan ((float) val));
+        FLOAT_OPS.put ("lessThanEquals", (attr, val) -> attr.lessThanEquals ((float) val));
+
+        TIMESTAMP_OPS.put ("eq", (attr, val) -> attr.eq ((Timestamp) val));
+        TIMESTAMP_OPS.put ("notEq", (attr, val) -> attr.notEq ((Timestamp) val));
+        TIMESTAMP_OPS.put ("isNull", (attr, val) -> attr.isNull ());
+        TIMESTAMP_OPS.put ("isNotNull", (attr, val) -> attr.isNull ());
+        TIMESTAMP_OPS.put ("in", (attr, val) -> attr.in (new HashSet ((List) val)));
+        TIMESTAMP_OPS.put ("notIn", (attr, val) -> attr.notIn (new HashSet ((List) val)));
+        TIMESTAMP_OPS.put ("greaterThan", (attr, val) -> attr.greaterThan ((Timestamp) val));
+        TIMESTAMP_OPS.put ("greaterThanEquals", (attr, val) -> attr.greaterThanEquals ((Timestamp) val));
+        TIMESTAMP_OPS.put ("lessThan", (attr, val) -> attr.lessThan ((Timestamp) val));
+        TIMESTAMP_OPS.put ("lessThanEquals", (attr, val) -> attr.lessThanEquals ((Timestamp) val));
+
+        ASOF_OPS.put ("eq", (attr, val) -> attr.eq ((Timestamp) val));
+        ASOF_OPS.put ("equalsEdgePoint", (attr, val) -> attr.equalsEdgePoint ());
+        ASOF_OPS.put ("equalsInfinity", (attr, val) -> attr.equalsInfinity ());
+
+        BOOLEAN_OPS.put ("eq", (attr, val) -> attr.eq ((boolean) val));
+        BOOLEAN_OPS.put ("notEq", (attr, val) -> attr.notEq ((boolean) val));
+        BOOLEAN_OPS.put ("isNull", (attr, val) -> attr.isNull ());
+        BOOLEAN_OPS.put ("isNotNull", (attr, val) -> attr.isNull ());
+    }
+
+
+    FilterQueryBuilder ()
+    {
+    }
+
+    private Operation defaultMilestoning (Operation op, RelatedFinder finder)
+    {
+        if (finder.getAsOfAttributes () == null) return op;
+        Timestamp now = new Timestamp (System.currentTimeMillis ());
+        for (AsOfAttribute each : finder.getAsOfAttributes ())
+        {
+            if (op.zGetAsOfOp (each) == null)
+            {
+                op = op.and (each.eq (now));
+            }
+        }
+        return op;

```

Comment:
'now' is not a good choice here. The xml defines a `defaultIfNotSpecified`, which should be honored here. `defaultIfNotSpecified` is not recommended for `businessDate`, because most of the time, not specifying it explicitly is a bug. This should be kept consistent. `now` is also pretty useless as a `businessDate`.